source/table.h

```diff
@@ -74,7 +74,7 @@ typedef struct spv_ext_inst_desc_t {
   const uint32_t ext_inst;
   const uint32_t numCapabilities;
   const spv::Capability* capabilities;
-  const spv_operand_type_t operandTypes[16];  // TODO: Smaller/larger?
+  const spv_operand_type_t operandTypes[64];  // TODO: Smaller/larger?
```

Comment:
I think the corresponding extended instruction set only needs 33 slots.
I suggest lowering this to 40.

The fixed array sizes are kind of ugly, a quick implementation tradefoff I made about 7 years go.  I'm long overdue making a more compact but still static and immutable data structure.  That's on me.

Reply:
I have lowered it to 40